README.md

```diff
@@ -5,7 +5,7 @@
 ## Overview
 
 An operator for leveraging NSX as the default container networking solution for an
-Openshift cluster. The operator will be deployed in the early phases of cluster
+Kubernetes/Openshift cluster. The operator will be deployed in the early phases of cluster
```

Comment:
When integrating with Kubernetes, is it mandatory  to deploy it in the first phases of cluster deployment? 
I don't want to nitpick on the README file! If I recall the implementation correctly, we may also be able to deploy the operator on a cluster where NCP is already running, is that correct?

Reply:
Yes, in kubernetes, we don't have to deploy the operator so early like openshift. And the operator can start on a cluster where NCP is already running, although it's very possible that the operator recreated NCP pod when it founds the configmap between the operator and the existing NCP are different.

Reply:
I improved the description but not sure if it is accurate enough, could you please look at it again?